src/docs/swagger.json

```diff
@@ -81,7 +81,7 @@
         }
       }
     },
-    "/password-reset": {
+    "/reset": {
```

Comment:
@truestbyheart I thought we've updated these URLs? Kindly sync with @nkpremices and @alainburindi and update this file accordingly.